app/main.py

```diff
@@ -1,13 +1,61 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    if len(error) == 0:
+        return {}
+    formatted_error = {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
+    return formatted_error
```

Comment:
```suggestion
    return {
        "line": error["line_number"],
        "column": error["column_number"],
        "message": error["text"],
        "name": error["code"],
        "source": "flake8"
    }
```
Please read task again: All functions must contain only the return keyword